src/main/java/uk/gov/companieshouse/accounts/filing/mapper/FilingGeneratorMapper.java

```diff
@@ -30,7 +31,12 @@ public FilingApi mapToFilingApi(AccountsFilingEntry accountsFilingEntry) {
         Map<String, String> descriptionValue = Collections.singletonMap("made up date", madeUpDate);
 
         var filingApiEntity = new FilingApi();
-        filingApiEntity.setDescription("Package accounts made up to " + formatMadeUpDate(madeUpDate));
+        if(PackageTypeApi.OVERSEAS == accountsFilingEntry.getPackageType()){
+            filingApiEntity.setDescription("Package accounts with package type overseas");
+        }else{
+            filingApiEntity.setDescription("Package accounts made up to " + formatMadeUpDate(madeUpDate));
+        }
```

Comment:
Could we fix the spacing here a bit
```suggestion
        if (PackageTypeApi.OVERSEAS == accountsFilingEntry.getPackageType()) {
            filingApiEntity.setDescription("Package accounts with package type overseas");
        } else {
            filingApiEntity.setDescription("Package accounts made up to " + formatMadeUpDate(madeUpDate));
        }
```